src/plots/cartesian/tick_label_defaults.js

```diff
@@ -40,6 +40,7 @@ module.exports = function handleTickLabelDefaults(containerIn, containerOut, coe
 
         if(axType !== 'category') {
             var tickFormat = coerce('tickformat');
+            coerce('tickformatstops');
```

Comment:
... also, you'll need to a little more work coercing `tickformatstop` after having addressed https://github.com/plotly/plotly.js/pull/1965/files#r134489590 - see the `layout.slider` [defaults](https://github.com/plotly/plotly.js/blob/ff8ecb1b6a63559a897c2ba744cecdef0c6660a8/src/components/sliders/defaults.js#L84) for an example.